build-tools/package.json

```diff
@@ -1,7 +1,8 @@
 {
-  "name": "neon-compute",
+  "name": "build-tools",
   "private": true,
   "dependencies": {
+    "@redocly/cli": "1.34.4",
```

Comment:
I assume this manifest is not really used to build prod code but can we still make clear intend by running npm install with `--save-dev` redocly shouldn't be a prod dependency and only needed for dev work

Reply:
Done